### PR TITLE
Adding module for buscoannotation/buscopainterlep. Closes #34.

### LINKED ISSUE #34: new module: buscoannotation/buscopainterlep
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by weaglesBio**
## PR checklist

Closes #34 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/buscoannotation/buscopainterlep/main.nf [hunk 1]
```diff
@@ -0,0 +1,52 @@
+process BUSCOANNOTATION_BUSCOPAINTERLEP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    def version = '0.001-c4'
```
Comment (gq1) [inline modules/buscoannotation/buscopainterlep/main.nf:5]: you have c5 new image now? do you want to update here?

**Comment by gq1** [comment]
Can you fix the md5 checking test?

**Comment by priyanka-surana** [comment]
The tests need to work for us to do a proper review. Does it work locally? Is this a GitHub CI issue?

File: modules/buscoannotation/buscopainterlep/meta.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      pattern: "versions.yml"
+  - complete_location_tsv:
+      type: file
+      description: TSV file
+      pattern: "*_complete_location.tsv"
+  - duplicated_location_tsv:
+      type: file
+      description: TSV file
+      pattern: "*_duplicated_location.tsv"
+  - duplicated_location_tsv:
```
Comment (priyanka-surana) [inline modules/buscoannotation/buscopainterlep/meta.yml:43]: You have 2 `duplicated_location_tsv`, but no `summary_tsv`. Also the description needs to tell the user what is expected. Please be more detailed.

**Comment by weaglesBio** [comment]
nf-core modules linting had no errors when run locally.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍

**Pull request merged**